Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollColliderTranslationManipulators.cpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzFramework/Physics/Character.h>
+#include <EMotionFX/CommandSystem/Source/ColliderCommands.h>
+#include <EMotionFX/Source/Actor.h>
+#include <EMotionFX/Source/Node.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/ColliderContainerWidget.h>
+#include <Editor/ObjectEditor.h>
+#include <Editor/Plugins/Ragdoll/RagdollColliderTranslationManipulators.h>
+
+namespace EMotionFX
+{
+    RagdollColliderTranslationManipulators::RagdollColliderTranslationManipulators()
+        : m_translationManipulators(
+              AzToolsFramework::TranslationManipulators::Dimensions::Three, AZ::Transform::CreateIdentity(), AZ::Vector3::CreateOne())
+    {
+        m_adjustColliderCallback = new DataChangedCallback(this, false);
+        EMStudio::GetCommandManager()->RegisterCommandCallback("AdjustCollider", m_adjustColliderCallback);
+    }
+
+    RagdollColliderTranslationManipulators::~RagdollColliderTranslationManipulators()
+    {
+        EMStudio::GetCommandManager()->RemoveCommandCallback(m_adjustColliderCallback, false);
+        delete m_adjustColliderCallback;
+    }
+
+    void RagdollColliderTranslationManipulators::Setup(RagdollManipulatorData& ragdollManipulatorData)
+    {
+        m_ragdollManipulatorData = ragdollManipulatorData;
+
+        if (!m_ragdollManipulatorData.m_valid || !m_ragdollManipulatorData.m_colliderNodeConfiguration ||
```

Comment:
nit: A guide comment here might be nice, what could cause these error conditions to occur for example?